content/rs/administering/monitoring-metrics/prometheus-metrics-definitions.md

```diff
@@ -224,10 +224,10 @@ Here are the metrics available to Prometheus:
 | redis_expired_keys | Keys expired so far (since restart) |
 | redis_forwarding_state | Shard forwarding state (on or off) |
 | redis_keys_trimmed | The number of keys that were trimmed in the current or last resharding process |
-| redis_keyspace_read_hits | Rate of read operations accessing an existing keyspace (ops/sec) |
-| redis_keyspace_read_misses | Rate of read operations accessing an non-existing keyspace (ops/sec) |
-| redis_keyspace_write_hits | Rate of write operations accessing an existing keyspace (ops/sec) |
-| redis_keyspace_write_misses | Rate of write operations accessing an non-existing keyspace (ops/sec) |
+| redis_keyspace_read_hits | Number of read operations accessing an existing keyspace |
```

Comment:
I don't care much but maybe you do.
In one place you use 'Number of' and in another `Running counter of the number of`
maybe you wanna make it consistent and use the shorter form for both..